### PR TITLE
Added a note to page

### PR DESCRIPTION
**Pull request by v-kikl**
@jgeurten, I added the note about separating allow and deny rules is at the end of the "More information about file path rules" section.

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md [hunk 1]
```diff
@@ -126,6 +126,9 @@ Wildcards can be used at the beginning or end of a path rule; only one wildcard
 
 You can also use the following macros when the exact volume may vary: `%OSDRIVE%`, `%WINDIR%`, `%SYSTEM32%`.
 
+>[!NOTE]
+>We recommend maintaining separate ALLOW and DENY policies on version 1903 and higher, if for no other reason than it makes it a bit easier for an average person to reason over the policy.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md:130]: Please include MarkDown indent marker compatibility spacing:
```suggestion
> [!NOTE]
> We recommend maintaining separate ALLOW and DENY policies on version 1903 and higher, if for no other reason than it makes it a bit easier for an average person to reason over the policy.
```
The last half of the sentence could use some phrasing improvement, but I am uncertain how to phrase it to clarify the intended meaning.

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ A Windows Defender Application Control (WDAC) policy logs events locally in Wind
 |--------|-----------|
 | 8028 | Audit script/MSI file generated by Windows LockDown Policy (WLDP) being called by the script hosts themselves. Note: there is no WDAC enforcement on third-party script hosts. |
 | 8029 | Block script/MSI file |
+| 8036| COM object was blocked. Learn more about COM object authorization: Allow COM object registration in a WDAC policy (Windows 10) - Windows security - Microsoft Docs |
```
Comment (jgeurten) [inline windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:44]: The Allow COM object registration .. is supposed to be a hyperlink

**Review by jgeurten — CHANGES_REQUESTED** [review]

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md (hunk 2 above)
Comment (illfated) [reply windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:44]: Thank you, found it, it is a parallel document in the same directory as this file:
```suggestion
| 8036 | COM object was blocked. Learn more about COM object authorization: [Allow COM object registration in a WDAC policy (Windows 10)](allow-com-object-registration-in-windows-defender-application-control-policy.md) |
```
(also: keep the same spacing as the line before & after)

Comment (illfated) [reply windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:44]: Links to the document in question:
- https://docs.microsoft.com/windows/security/threat-protection/windows-defender-application-control/allow-com-object-registration-in-windows-defender-application-control-policy
- https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/windows-defender-application-control/allow-com-object-registration-in-windows-defender-application-control-policy.md

File: windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md [hunk 3]
```diff
@@ -126,6 +126,9 @@ Wildcards can be used at the beginning or end of a path rule; only one wildcard
 
 You can also use the following macros when the exact volume may vary: `%OSDRIVE%`, `%WINDIR%`, `%SYSTEM32%`.
 
+>[!NOTE]
+>For others to better understand the WDAC policies that has been deployed, we recommend maintaining separate ALLOW and DENY policies on version 1903 and higher.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md:130]: Looks like the suggested change got skipped. May we have the standard MarkDown indent marker compatibility spacing?
```suggestion
> [!NOTE]
> For others to better understand the WDAC policies that has been deployed, we recommend maintaining separate ALLOW and DENY policies on version 1903 and higher.
```

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md [hunk 4]
```diff
@@ -41,6 +41,7 @@ A Windows Defender Application Control (WDAC) policy logs events locally in Wind
 |--------|-----------|
 | 8028 | Audit script/MSI file generated by Windows LockDown Policy (WLDP) being called by the script hosts themselves. Note: there is no WDAC enforcement on third-party script hosts. |
 | 8029 | Block script/MSI file |
+| 8036| COM object was blocked. Learn more about COM object authorization: [Allow COM object registration in a WDAC policy (Windows 10) - Windows security - Microsoft Docs](allow-com-object-registration-in-windows-defender-application-control-policy). |
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:44]:
```suggestion
| 8036| COM object was blocked. To learn more about COM object authorization, see [Allow COM object registration in a Windows Defender Application Control policy](/windows/security/threat-protection/windows-defender-application-control/allow-com-object-registration-in-windows-defender-application-control-policy). |
```

File: windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md [hunk 5]
```diff
@@ -108,7 +109,7 @@ A list of other relevant event IDs and their corresponding description.
 | 3082 | If the policy was in enforced mode, the non-WHQL driver would have been denied by the policy. | 
 | 3084 | Code Integrity will enforce the WHQL Required policy setting on this session. |
 | 3085 | Code Integrity will not enforce the WHQL Required policy setting on this session. |
-| 3086 | COM object was blocked. Learn more about COM object authorization: Allow COM object registration in a WDAC policy (Windows 10) - Windows security - Microsoft Docs|
+| 3086 | The file under validation does not meet the signing requirements for an IUM (isolated user mode) process. |
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/event-id-explanations.md:112]:
```suggestion
| 3086 | The file under validation does not meet the signing requirements for an isolated user mode (IUM) process. |
```

File: windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md [hunk 6]
```diff
@@ -126,6 +126,9 @@ Wildcards can be used at the beginning or end of a path rule; only one wildcard
 
 You can also use the following macros when the exact volume may vary: `%OSDRIVE%`, `%WINDIR%`, `%SYSTEM32%`.
 
+> [!NOTE]
+> For others to better understand the WDAC policies that has been deployed, we recommend maintaining separate ALLOW and DENY policies on version 1903 and higher.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/select-types-of-rules-to-create.md:130]:
```suggestion
> For others to better understand the WDAC policies that have been deployed, we recommend maintaining separate allow and deny policies on Windows 10, version 1903 and later.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @v-kikl, please commit my suggested changes if you agree with them. Many thanks.

**Comment by v-kikl** [comment]
Is this PR ready to be approved? Or, are there other changes that need to be made? @JohanFreelancer9?

**Comment by JohanFreelancer9** [comment]
@v-kikl - it does not seem as if my suggested changes have been committed. Could you please commit them?

**Review by jgeurten — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**